batch/batch/driver/instance_collection.py

```diff
@@ -133,18 +134,28 @@ async def check_on_instance(self, instance):
 
         # PROVISIONING, STAGING, RUNNING, STOPPING, TERMINATED
         gce_state = spec['status']
+
         log.info(f'{instance} gce_state {gce_state}')
 
+        if (gce_state == 'PROVISIONING'
+                and instance.state == 'pending'
+                and time_msecs() - instance.time_created > 5 * 60 * 1000):
+            log.info(f'{instance} did not provision within 5m after creation, deleting')
```

Comment:
I think this should be an error, it will cause us to respond to high rates of long-provisioning instances.